### ISSUE TITLE
Loss function returns a vector rather than scalar tensor

### ISSUE THREAD

**Issue by masoud-moghini** [ZhaoJ9014/face.evoLVe.PyTorch#205]
Loss functions' dictionary has been declared in   (https://github.com/ZhaoJ9014/face.evoLVe/blob/a9897bda52bdbb8d7c2fe28f1e21827dfd69d14e/train.py#L113)
then the amount returned by these functions has been used in(https://github.com/ZhaoJ9014/face.evoLVe/blob/a9897bda52bdbb8d7c2fe28f1e21827dfd69d14e/train.py#L202) to update losses during training 
But the problem is that loss functions return vector of logits rather than their CrossEntropyLoss
